frontends/web/src/routes/account/account.tsx

```diff
@@ -257,21 +248,20 @@ class Account extends Component<Props, State> {
             accounts,
         }: RenderableProps<Props>,
         {
+            status,
             transactions,
-            accountSynced,
-            offlineMode,
             balance,
             hasCard,
             exported,
             accountInfo,
-            fatalError,
             syncedAddressesCount,
         }: State) {
         const account = accounts &&
                         accounts.find(acct => acct.code === code);
-        if (!account) {
+        if (!account || status === undefined) {
             return null;
         }
```

Comment:
FYI, when new-api is merged (soon™) I'd like to get rid of this `return null` because that is the main reason for the flickering when changing accounts, I believe. The alternative is to let all components render, but they might render them self invisible, or with a gray area to indicate that something is comming.